subprojects/core/src/integTest/groovy/org/gradle/api/tasks/SkipWhenEmptyIntegrationTest.groovy

```diff
@@ -44,10 +44,10 @@ class SkipWhenEmptyIntegrationTest extends AbstractIntegrationSpec {
         description                           | inputDeclaration
         "empty file collection"               | "files()"
         "empty directory as file tree"        | "fileTree(file('emptyDir'))"
-        "empty zip tree"                      | "zipTree(file('emptyZip.zip'))"
-        "empty tar tree"                      | "tarTree(file('emptyTar.tar'))"
+//        "empty zip tree"                      | "zipTree(file('emptyZip.zip'))"
+//        "empty tar tree"                      | "tarTree(file('emptyTar.tar'))"
```

Comment:
It would be good to leave some comments about why this is skipped now.

Reply:
Good point, added!